intermediate_notebooks/E2E/mortgage/mortgage_e2e.ipynb

```diff
@@ -107,8 +152,8 @@
     "acq_data_path = \"/path/to/mortgage/acq\"\n",
```

Comment:
Can you update this cell to behave like the 2nd cell of the weather notebook?

It'd be great if it could define:
`data_dir = '/data/mortgage'`

And then check whether the datafiles already exist in that directory. If they don't already exist in data_dir, then the script will automatically download the relevant datafiles.

Reply:
Made simple logic updates. Don't check individual files since we cannot download (do not provide on docs.rapids.ai) individual files right now.

Reply:
@taureandyernv how can we get this mortgage data into s3 as individual files too?